enrollment-server-onboarding/src/main/java/com/wultra/app/onboardingserver/controller/api/OnboardingController.java

```diff
@@ -98,6 +98,7 @@ public ObjectResponse<OnboardingStartResponse> startOnboarding(@EncryptedRequest
             throw new PowerAuthEncryptionException("Invalid request received during onboarding");
         }
 
+        // Onboarding process is locked inside the service
```

Comment:
dtto I suggest to move it to the to the javadoc.

Reply:
I will use the debug logging.